### PR TITLE
Remove debug print in a Parquet test

### PR DESCRIPTION
**Pull request by vuule**
## Description
Removed a debug print. That's it.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

**Review by karthikeyann — APPROVED** [review]

File: cpp/tests/io/parquet_test.cpp [hunk 1]
```diff
@@ -52,6 +52,9 @@
 #include <random>
 #include <type_traits>
 
+// Set this to true to enable printing, for debugging.
+auto constexpr print_enabled = false;
```
Comment (ttnghia) [inline cpp/tests/io/parquet_test.cpp:56]: Should this be read from env variable?

File: cpp/tests/io/parquet_test.cpp [hunk 2]
```diff
@@ -6534,7 +6537,7 @@ TEST_F(ParquetReaderTest, FilterFloatNAN)
   auto col0 = cudf::test::fixed_width_column_wrapper<float>(elements, elements + num_rows);
   auto col1 = cudf::test::fixed_width_column_wrapper<double>(elements, elements + num_rows);
 
-  cudf::test::print(col0);
+  if constexpr (print_enabled) { cudf::test::print(col0); }
```
Comment (davidwendt) [inline cpp/tests/io/parquet_test.cpp:6540]: Is this line even necessary? Seems this was added for debugging something specific and was just forgotten.
It does not seem difficult to re-add the line if debugging is needed in the future.
Also, future debugging would probably want to print a different variable in a different place and enabling the constant could produce confusing output.

Comment (karthikeyann) [reply cpp/tests/io/parquet_test.cpp:6540]:
```suggestion
```

File: cpp/tests/io/parquet_test.cpp (hunk 1 above)
Comment (vuule) [reply cpp/tests/io/parquet_test.cpp]: I copied the mechanism from structs_tests. Either way, this is removed now :)

**Review by davidwendt — APPROVED** [review]

**Review by ttnghia — APPROVED** [review]

**Pull request merged**